src/workloads/scale/LLTQuery.yml

```diff
@@ -0,0 +1,421 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  Workload to Benchmark the effect of LongLivedTransactions on an Update workload.
+
+GlobalDefaults:
```

Comment:
I assume a lot of this is the same in the other workloads? It's a shame we can't pull out that commonality.